### ISSUE TITLE
[Bug] empty-lines-between-test-cases recognizes comment lines as EmptyLine instances

### ISSUE THREAD

**Issue by polewczakp** [MarketSquare/robotframework-robocop#513]
### What happened?

Hi,
I added comments to separate visually some test cases from others, e.g.

```
# A
Test case A1
Test case A2

# B
Test case B1
Test case B2
```

Unfortunately I am getting an error:
"[W1004] empty-lines-between-test-cases: Invalid number of empty lines between test cases (%d/%d)"
and it points to the lines that match # A and # B.

It seems that it's checked via these lines:
```
if isinstance(token, EmptyLine):
    empty_lines += 1
```

Perhaps using something like this:
```
if not token.strip():
    empty_lines += 1
```
would solve the problem?

Best regards,
Pawel

### Robocop version

1.11.2

**Comment by polewczakp** [comment]
That's correct, it's templated. So it's the last case most probably.

**Comment by mnojek** [comment]
Thanks @polewczakp for the report and sorry that it took so long to fix it. It should be released with next version 2.1.0 soon.
